src/components/shared/forms/styles/toggleBtn.scss

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/themes';
+@import 'src/styles/mixins';
+.toggle-btn-div {
+  .pcs-toggle {
+    display: inline-block;
+    svg {
+      height: 15px;
+      cursor: pointer;
+      &.disabled { cursor: auto; }
+    }
+  }
+
+  @include themify($themes) {
+    .cls-1  {
+        stroke: themed('colorEnableToggleBackground');
+      }
+
+     .enable-toggle .cls-2, .disable-toggle .cls-2 {
+      fill: themed('colorToggle');
+      stroke: themed('colorToggle');
+    }
+
+    .disable-toggle, .disabled {
+      fill: themed('colorDisableToggleBackground');
```

Comment:
Do the toggle button colors really need to be different than the ones used for Btn? The Btn colors should all match up with Caitlin's redlines for buttons. Of course, you might have started this before those were in the theme file, but they should be there now.

Reply:
The toggle "button" is really more of a switch image, it's not covered by her generic buttons